src/index.html

```diff
@@ -1,12 +1,326 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="moyo-header">
+      <a href="#" class="logo-link">
+        <img
+          src="images/logo.png"
+          alt="moyo-logo"
+          class="logo-img"
+        >
+      </a>
+      <nav class="moyo-nav">
+        <a href="#" class="nav-links is-active">apple</a>
```

Comment:
```suggestion
        <a href="#" class="nav__link is-active">apple</a>
```
https://ru.bem.info/methodology/quick-start/
-використовуй бем 
-це одна лінка чому links